requirements/base.txt

```diff
@@ -1,3 +1,4 @@
 Django>=4.0,<4.1
 wagtail>=3.0,<3.1
 django-manifest-loader
+djangorestframework
```

Comment:
As we discussed in the meeting, we should add the version specifier here.

Reply:
:+1: